python/tvm/testing/utils.py

```diff
@@ -1513,3 +1514,8 @@ def identity_after(x, sleep):
 def terminate_self():
     """Testing function to terminate the process."""
     sys.exit(-1)
+
+
+def main():
+    test_file = inspect.getsourcefile(sys._getframe(1))
+    sys.exit(pytest.main([test_file] + sys.argv[1:]))
```

Comment:
just a nit but do we actually need to determine and pass in the test file name? [the examples](https://docs.pytest.org/en/7.1.x/how-to/usage.html#calling-pytest-from-python-code) don't seem to need it
```suggestion
    sys.exit(pytest.main(sys.argv[1:]))
```

Reply:
I think running without a specified test file will cause pytest to search the current file for tests.